androidApp/build.gradle.kts

```diff
@@ -2,8 +2,10 @@ plugins {
     id("com.android.application")
     kotlin("android")
     id("kover")
+    id("com.google.gms.google-services")
```

Comment:
We should add these plugins to `Plugin.kt` in `BuildSrc`. What do you think?

Reply:
Updated in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/82/commits/4f10e1f310cd5b7eb2e252678c8a22dc7d4d6afc